app/assets/stylesheets/admin/openfoodnetwork.css.scss

```diff
@@ -83,6 +83,8 @@ a.action--disabled {
 }
 
 form.order_cycle {
+  padding-bottom: 50px;
```

Comment:
I had to put this in because semarphoreci was consistently failing around this area. Upon inspection I found that in the admin there was a bit of button/div overlap and selenium was not able to click on a button. Let me try removing this and see if it works now. But yes, this is unrelated to the PR

Reply:
took it out and it's working fine. Not sure of the science behind it 😬